packages/gatsby-plugin-offline/src/gatsby-node.js

```diff
@@ -70,12 +76,12 @@ exports.onPostBuild = (args, pluginOptions) => {
     globDirectory: rootDir,
     globPatterns,
     modifyUrlPrefix: {
-      rootDir: ``,
+      [rootDir]: ``,
```

Comment:
Why do we need this?

Reply:
Initially this was added two years ago ([link](https://github.com/gatsbyjs/gatsby/commit/45b36270cd51220d26367df9bcf07ae97e9f6eac#diff-b4eb68a70122d3cae2435fe21a435ca9R21)) so I just kept it, but I think we don't actually need it - as you pointed out earlier, I implemented it wrongly with the Workbox config and nothing broke, so it seems as if we don't need it after all. I'll push a commit removing this line :)

Reply:
Perhaps at some point this was necessary, but then when the build system changed it was never removed, so it might've been useless for a long time before I submitted my first PR to Gatsby.